### DISCUSSION TITLE
Respect for geany

### DISCUSSION THREAD

**Discussion by goyalyashpal** [geany/geany#3421] [General]
I was just using geany today (mainly for compiling; for editing, 'was using Kate just for taking benefit of clangd LSP) and 
- pressed `Ctrl - /` by mistake. and huh... word navigation back.
- what next i wondered, `Ctrl - \` next word nice
- `Ctrl - ]` and `Ctrl - [` for what? para-wise (i.e. broken at empty line) navigation i guess 🤔 ?
- `Ctrl - b` for start/end of parenthesis?

woww! i only knew of `ctrl - arrow` keys lol. how many more navigation shortcuts do geany have - any cheatsheet?

Here: Go to keybinds https://www.geany.org/manual/current/index.html#go-to-keybindings (easily reachable from geany official homepage)

just early today i browser the editor color scheme themes (<tt>***V***iew > Change ***C***olor Scheme</tt>, not UI themes) - and woww, geany has some of best dark color schemes out of the box in my opinion. I picked the following ones:

```
oblivion 2 (my current), sleepy pastel, zenburn, 
bespin, kugel
```

I already admired geany from long ago due to its amazzzing adherence to _KIS_ principle 
- it's preferences are always sooo simple despite providing options for sooo many things (even for which explorer to choose). 
- It's build & run menus put even vscode to shame: it's flexible, versatile, and _just works_ - and never made me feel the lack of terminal (i am on windows)
- It's message window is sooo thoughtful - the scribble pad, and messages wow, no other editors has that. 
- its default UI theme is sooooo pleasant to eyes and mind - despite being light mode, it's balanced, it's colorful; not the dull boring BLINDING white every-modern-where else.
- even commenting distinction (for code/non-code) was thought about (`//~` vs `//`)

i have used lots of other editors (notepad++, vs codium, kate, notepad3)  - but there's something in geany which always i find myself to be lacking in other editors. It gives _Freedom_ (which corporate produce, despite being OSS, just can't give), _balance_ (which so much offerings lack), and _ingenuity_ through simple things. Other moodern editors are just based on technology, geany is based on problem solving thoughtful ideas.

and all that too in just 27 MB (for Geany 1.38) (Kate: 67 MB, VSCodium: 80-95 MB)

There are some missing things in geany too - pro'lly biggest of 'em being lack of LSP support. Other one being this issue: https://github.com/geany/geany/issues/3319 - Rest is all bearable. 

And a small nitpick i have (i am not saying to change it, just bringing in notice): 
a jin, or geany is not considered something good in this side of the world, i don't want any ghost or any wandering spirit with me. so, i _wish_ there could have been a better name. But it's ohkay.

Thanks a lot for providing such a simple yet super customisable, perfect BALANCE of the things editor. Lots of love.

**Comment by elextr** [comment]
Thank you for feedback, there is not enough, especially positive feedback :-)

> any cheatsheet?

`Help->Keyboard Shortcuts` gives keybindings in Geany (your ctrl+slash and backslash and ctrl+b are there) but that does not include keybindings that Scintilla sets by default, since Geany does not know about them, some are listed in the Geany manual, but not all, and of course they may change since Scintilla is a separate project.

**Comment by elextr** [comment]
Just a note, I deleted the extra copy of the discussion.

**Reply by goyalyashpal** [reply]
hehe, the same issue happened from ur side lolol:

![image](https://user-images.githubusercontent.com/19423063/222892048-1c71e8a0-ab68-42e1-9fae-19fbbc5113a5.png)

**Comment by elextr** [comment]
Yeah, I got a "You can't comment at the moment" red message the first time I submitted, the second time it worked, but weirdly it only shows once on the discussion here, not twice?

**Reply by goyalyashpal** [reply]
bug in the internet websites lol - can't do much about it :)
it showed that comment twice only for some minutes, now it's only showing it once. i originally thought u deleted the other copy, but seems not. so, maybe github recovered.

**Reply by elextr** [reply]
Probably did the first submit partly, and updated your side, then completed the second submit properly, and your side showed two until it updated again.
